### ISSUE TITLE
Viewport Driver Crash: StructRNA of type CGC_Polypen has been removed

### ISSUE THREAD

**Issue by MasterJames** [CGCookie/retopoflow#470]
Often when in Polypen and I click the wrong entity then is expected I'm assuming it might be the fourth point is parallel to others in the triangle. Well it totally kills the Viewport. You can save and recover luckily but it crashes the viewport until a full restart of Blender.

```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Me\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\retopoflow-master\modaloperator.py", line 172, in draw_callback_postpixel self.handle_exception()
  File "C:\Program Files\Blender Foundation\Blender\2.79\scripts\modules\bpy_types.py", line 601, in __getattribute__ properties = StructRNA.path_resolve(self, "properties")
ReferenceError: StructRNA of type CGC_Polypen has been removed
```

In hindsight I think maybe it does scramble some things around a little. Possible some things that are not paired to the surface of the guide shape. For instance extrusion in nose or mouth. Then again that might have nothing to do with it.

**Comment by vxlcoder** [comment]
Thanks for the report, @MasterJames.

Similar to #473, can you post a screenshot of what you're describing in text?

Also, is this RetopoFlow 1.3 or 2.0 beta?

Thanks!

**Comment by MasterJames** [comment]
It's version 1.3.2
screenshot's maybe if when it happens again etc. it just messes up and puts a black square I guess at 0,0 so it hovers overlapping etc. GPU crash I guess.

**Comment by vxlcoder** [comment]
I believe this issue applies to v1.x only, so I'm going to close this issue now that v2.0 is officially released (see `release/2.0.0` branch or Blender Market page).  If this is still an issue with v2.0, please reopen, and post more details.  Thanks for reporting this issue, and we look forward to seeing how v2.0 helps your retopo workflow!
